### DISCUSSION TITLE
restart error: start a steady simulation from a saved solution(.csv file)

### DISCUSSION THREAD

**Discussion by songjuzheng** [su2code/SU2#1936] [Q&A]
I'm learning to start a steady simulation from a saved solution(solution_flow.csv). But I get the following error:
>>Error in "void CSolver::Read_SU2_Restart_Binary(CGeometry*, const CConfig*, std::__cxx11::string)":
>>Unable to open SU2 restart file solution_flow.dat

Some of the keywords in my cfg file are as follows：
SOLVER= RANS
KIND_TURB_MODEL= SST
MATH_PROBLEM= DIRECT
RESTART_SOL= YES

SOLUTION_FILENAME= solution_flow.csv
OUTPUT_FILES = (RESTART_ASCII, TECPLOT_ASCII, SURFACE_CSV)
RESTART_FILENAME= restart_flow.csv

**Comment by WallyMaier** [comment]
Hi,
So from your post, you should have a restart_flow.csv in your folder.
What you will want to do:
1.) Rename restart_flow.csv to solution_flow.csv
2.) Add: READ_BINARY_RESTART= NO  to your config to read the csv files.

**Reply by songjuzheng** [reply]
Thanks！Step 2 is missing from the tutorial.

**Reply by bigfooted** [reply]
Hi, which tutorial is it? Then I can add that information to the page.

**Reply by songjuzheng** [reply]
> Hi, which tutorial is it? Then I can add that information to the page.

https://su2code.github.io/docs_v7/Restart-File/
